### PR TITLE
docs: Update stories to include custom styles

### PR DESCRIPTION
**Pull request by RayRedGoose**
## Summary

Resolves: #2728  

Custom Styles examples have been added to refactored component stories.

## Release Category
Documentation

### CODE REVIEW

**Comment by mannycarrera4** [comment]
removed ready for review until we're able to update stories for new pattern!

File: modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx [hunk 1]
```diff
@@ -654,16 +654,6 @@ const MyComponent = StyledRadioButton('div')({
...
-- `width` and `height` props has been deprecated and will be removed in a future version. Consider
-  using `cs` props instead to set svg sizes.
-
-```tsx
-// before
-<Graphic icon={myGraphic} width={160} />
-
-// after
-<Graphic icon={myGraphic} cs={{"& svg": {width: '10rem'}}} />
-```
```
Comment (mannycarrera4) [inline modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx:666]: i would keep this in here?

File: modules/styling/stories/HowToStyle.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
+import {CustomButton} from './examples/CustomButton';
```
Comment (mannycarrera4) [inline modules/styling/stories/HowToStyle.stories.mdx:1]: General comment for this file. Is this a `How to Customize Styles` with our components, or `How to Style` using our utilities. Most people want to know how to style our components. We can probably save `How to Style` for something like the css kit where we highlight using the utilities with basic dom nodes. I think for this release we should focus on how to style with our components.

File: modules/css/badge.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+@keyframes animation-eec5d2 {
+  from {
+    transform: scale(0.85);
```
Comment (alanbsmith) [inline modules/css/badge.css:3]: Did you want to commit the CSS files?

File: modules/react/button/stories/button/examples/CustomStyles.tsx [hunk 4]
```diff
@@ -1,85 +1,49 @@
...
 
-const getDropdownColors = () => {
-  return {
-    default: {
-      background: colors.berrySmoothie400,
-      icon: colors.frenchVanilla100,
-      label: colors.frenchVanilla100,
+const myButtonStencil = createStencil({
+  base: {
+    [buttonStencil.vars.background]: system.color.static.green.soft,
```
Comment (josh-bagwell) [inline modules/react/button/stories/button/examples/CustomStyles.tsx:13]: Are we supposed to be using the `static` object here or the `bg` object?

File: modules/styling/stories/HowToStyle.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+There are multiple ways to customize styles for our components within Canvas Kit. What approach you
+choose depends on your use case. Ranging from some simple overrides to fully custom solutions, here
+are the following options.
+
+- [Styles](#styles)
+- [Stencils](#stencils)
+
+## Styles
+
+### Use `createStyles` With `cs` If
```
Comment (josh-bagwell) [inline modules/styling/stories/HowToStyle.stories.mdx:18]:
```suggestion
### Use `createStyles` with `cs` if:
```

File: modules/styling/stories/HowToStyle.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+})
+//...
+<Text cs={uppercaseTextStyles}>My uppercased text</Text>;
+```
+
+> **Note:** `createStyles` handles wrapping our token variables in `var(--${tokenName})`
+
+## Stencils
+
+### Use Stencils If
```
Comment (josh-bagwell) [inline modules/styling/stories/HowToStyle.stories.mdx:41]:
```suggestion
### Using Stencils
```

File: modules/styling/stories/HowToStyle.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+extraction into CSS files.
+
+```tsx
+export const buttonStencil = createStencil({
+  vars: {
+    backgroundColor: '',
+    textColor: '',
+  },
+  base: ({backgroundColor, textColor}) => ({
+    backgroundColor: cssVar(backgroundColor, base.berrySmoothie400),
```
Comment (josh-bagwell) [inline modules/styling/stories/HowToStyle.stories.mdx:54]: Do these need `cssVar`?

File: modules/styling/stories/HowToStyle.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+      </Element>
+    );
+  },
+});
+```
+
+> **Note:** We're using `handleCsProp` which handles the merging of `className`, `style` and `cs`
+> prop correctly.
+
+## Stencils For Reusable Components
```
Comment (josh-bagwell) [inline modules/styling/stories/HowToStyle.stories.mdx:93]:
```suggestion
## Stencils for Reusable Components
```

File: modules/styling/stories/HowToStyle.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+Stencils allow you compose component styles that need to handle modifiers (variants) while also
+exposing variables that can be modified.
```
Comment (josh-bagwell) [inline modules/styling/stories/HowToStyle.stories.mdx:96]:
```suggestion
Stencils allow you to compose component styles that need to handle modifiers (variants) while also
exposing variables that can be modified.
```

File: modules/styling/stories/examples/CustomIcon.tsx [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+import {createStencil, handleCsProp} from '@workday/canvas-kit-styling';
+import {systemIconStencil} from '@workday/canvas-kit-react/icon';
+import {system} from '@workday/canvas-tokens-web';
+
+const myIconStencil = createStencil({
+  extends: systemIconStencil,
+  base: {
+    [systemIconStencil.vars.color]: system.color.icon.primary.default,
+    [systemIconStencil.vars.accentColor]: system.color.icon.critical.default,
+    [systemIconStencil.vars.size]: '1rem',
```
Comment (josh-bagwell) [inline modules/styling/stories/examples/CustomIcon.tsx:11]:
```suggestion
    [systemIconStencil.vars.size]: system.space.x4,
```

File: modules/styling/stories/HowToStyle.stories.mdx (hunk 7 above)
Comment (RayRedGoose) [reply modules/styling/stories/HowToStyle.stories.mdx:54]: yes, they need it as fallback for variable

Comment (mannycarrera4) [reply modules/styling/stories/HowToStyle.stories.mdx:54]: this example doesn't extend `buttonStencil` so there's no default. So we either add the extends and remove this, or keep as is.

Comment (josh-bagwell) [reply modules/styling/stories/HowToStyle.stories.mdx:54]: Thanks for the clarification!

File: modules/react/button/stories/button/examples/CustomStyles.tsx (hunk 4 above)
Comment (josh-bagwell) [reply modules/react/button/stories/button/examples/CustomStyles.tsx:13]: This was discussed in stand. Static is to be used.

**Review by mannycarrera4 — APPROVED** [review]

**Review by josh-bagwell — APPROVED** [review]

**Pull request merged**